tools/train_net.py

```diff
@@ -11,22 +11,25 @@
 import os
 
 import torch
+
 from maskrcnn_benchmark.config import cfg
 from maskrcnn_benchmark.data import make_data_loader
-from maskrcnn_benchmark.solver import make_lr_scheduler
-from maskrcnn_benchmark.solver import make_optimizer
 from maskrcnn_benchmark.engine.inference import inference
 from maskrcnn_benchmark.engine.trainer import do_train
 from maskrcnn_benchmark.modeling.detector import build_detection_model
+from maskrcnn_benchmark.solver import make_lr_scheduler
+from maskrcnn_benchmark.solver import make_optimizer
 from maskrcnn_benchmark.utils.checkpoint import DetectronCheckpointer
 from maskrcnn_benchmark.utils.collect_env import collect_env_info
 from maskrcnn_benchmark.utils.comm import synchronize, get_rank
 from maskrcnn_benchmark.utils.imports import import_file
 from maskrcnn_benchmark.utils.logger import setup_logger
+from maskrcnn_benchmark.utils.metric_logger import (
+    MetricLogger, TensorboardLogger)
```

Comment:
nit: if we are going to use 2 lines, why not to use the same style used in the `solver` imports (L20-21)? Also, IMO, it would be nice if you could rename `TensorboardLogger` to `TensorBoardLogger` :laughing:

```python
from maskrcnn_benchmark.utils.metric_logger import MetricLogger
from maskrcnn_benchmark.utils.metric_logger import TensorBoardLogger
```

Reply:
why would you separate imports from the same module?

Reply:
I wouldn't, but this would be the only exception in this repository, if I am not wrong. I have no problem with multi-line imports ([PEP 328](https://www.python.org/dev/peps/pep-0328/)), but I do with inconsistencies :smile: